src/components/login-view/login-view.jsx

```diff
@@ -14,7 +14,7 @@ export const LoginView = ({ onLoggedIn }) => {
       Password: password
     };
 
-    fetch("https://my-flix-host.onrender.com/movies", {
```

Comment:
- The url should be `https://my-flix-host.onrender.com/login` for the login-view component.

Reply:
- The URL should be `https://my-flix-host.onrender.com/users` for the signup component